app/2.0/docs/devguide/custom-css-properties.md

```diff
@@ -488,7 +488,7 @@ The custom properties shim doesn't support styling distributed elements.
 
 ```css
 /* Not supported */
-:host ::content div {
+:host ::slotted(*) div {
```

Comment:
I believe this is invalid as nothing should be to the right of `::slotted()`, I'm pretty sure the correct syntax would be `::slotted(div)`.